package.json

```
@@ -3,6 +3,7 @@
   "version": "8.8.1",
   "description": "React Bootstrap 4 components",
   "main": "lib/index.js",
+  "types": "types/index.d.ts",
```

```suggestion
  "types": "es/index.d.ts",
```

This should point to the dts file in the release package.